src/partials/head-scripts-lightbend-cloudstate.hbs

```diff
@@ -1,2 +1,13 @@
 <!--Head Scripts Lightbend Cloudstate-->
-<script src="https://cdn.cookielaw.org/consent/5315597d-10f5-4faa-a8d0-66d98af23365.js" type="text/javascript" charset="UTF-8"></script>
\ No newline at end of file
+<script>
+  var head = document.getElementsByTagName('head')[0];
+  if (!window.location.href.startsWith("file://")) {
```

Comment:
Maybe better if we target `localhost`. At least in our case, we have a `make` tasks that starts  a python http server on `localhost:8000`. 

On the other hand, maybe we should not add code to solve it, but have it fixed locally. I added `cdn.cookielaw.org` to `/etc/hosts` and that's good enough for now.

Reply:
Updated adding `localhost` to the filtered urls, I was wondering about this use-case! 👍 

I'm totally fine in not merging this if it's not necessary, but, IMHO, this ease the local development a lot compared to the complexity it's introducing. I would not be extremely happy to apply global, machine-wide configurations to work around the issue :-(

Reply:
👍, you are right about tweaking local environment. Not a good idea.